cpp/order/Steinhardt.h

```diff
@@ -122,6 +122,24 @@ class Steinhardt
         return m_norm;
     }
 
+    //!< Whether to take a second shell average
+    bool isAverage()
+    {
+        return m_average;
+    }
+
+    //!< Whether to use the third-order invariant Wl
+    bool isWl()
+    {
+        return m_Wl;
+    }
+
+    //!< Whether to use neighbor weights in computing Qlmi
+    bool isWeighted()
+    {
+        return m_weighted;
+    }
+
```

Comment:
Do we actually need these? They're not even exposed in the Cython API. This is related to #266, I think we need further discussion there on whether or not we want to consistently expose all constructor arguments via properties. I think it's not a bad thing, but we should be more consistent about it. In this case, please either remove these methods or add properties to the class that use them.

Reply:
I exposed the properties and added them to the repr and plot labels.